assets/aml-benchmark/tests/test_prompt_crafter_checksum.py

```diff
@@ -0,0 +1,169 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+"""Tests for Prompt Crafter Checksums for various samples."""
+
+import os
+import sys
+import pytest
+
+from .test_utils import get_src_dir
+
+sys.path.append(get_src_dir())
+try:
+    from prompt_crafter.package.prompt_crafter import PromptCrafter
+except ImportError:
+    raise ImportError("Please install the package 'prompt_crafter' to run this test.")
```

Comment:
Is this something that we really need? Line 12 should be sufficient to handle this import.

Reply:
Agreed but the reason I put this is the code_health fails with E402 module level import not at top of file. Hence, this try except logic was added.